### ISSUE TITLE
Mega Menus - ACL issue

### ISSUE THREAD

**Issue by phuongnd** [cleversoft/Zo2-Joomla-Template-Framework#284]
Some customers report the following issue: 
-  I have a question for the MegaMenu functionality: I have some menu items that should be visible only for registered users. In the side menu module works fine, but the MegaMenu completely ignores those setting. Is there a way for the MegaMenu to consider these ACL restrictions.
- Have an issue where The MegaMenu used in the zo2_marco template ignores Joomla ACL.
  Basically want to put Login/Logout menu items in the menu, and have the Login menu display to "guest" users, the Logout menu display to "Registered" users.
  Joomla ACL settings work in other menus, but not the MegaMenu.
